packages/dd-trace/src/appsec/sdk/user_blocking.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
```

Comment:
In the most of the files, 'use strict' is followed by a white line.
```suggestion
'use strict'

```